### PR TITLE
Adding README.md on how to contribute to the project

### PR DESCRIPTION
**Pull request by filipemcarvalho**
Also: Moved CFS page in the folder structure

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -54,21 +55,20 @@ Check our events <a href="https://portotestersmeetup.eventbrite.com" target="_bl
...
-</div>
+  <div id="text">Licenciado em Informática de Gestão e 19 anos de experiência profissional em IT onde em 2008 enveredou na área de Testing e sua carreira progrediu naturalmente de junior a QA Team Lead seguido de Test Manager. Actualmente o seu desafio é de implementar test strategy, processos e procedimentos na entrega de produtos web na VentureOak com os padrões de qualidade elevados.</div>
 </div>
 
 # Call for Speakers
 
-We are always looking for speakers for these events. If you want to talk and share some of your knowledge, please let us know and fill <a href="/PTM_PAGE/CFS">this small form</a>
+We are always looking for speakers for these events. If you want to talk and share some of your knowledge, please let us know and fill <a href="/PTM_PAGE/pages/CFS">this small form</a>
+
+# Want to contribute?
```
Comment (FabioCB) [inline index.md]: Contribute for What? we should have specified that is for the website

**Review by FabioCB — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+5. Select where your site is published - should be under something like https://github.com/yourgithubusername/PTM_PAGE
+
+# First time open-source?
+
+Here is some guidance: https://opensource.guide/how-to-contribute/
+
+# Contributors
+
+- Diogo Junior
+- Fabio Barbosa
```
Comment (FabioCB) [inline README.md]: É Fábio Barbosa

**Review by FabioCB — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+# How to contribute
+
+If you want to contribute to a project and make it better, your help is very welcome.
+
+1. Fork the repository https://github.com/portotestersmeetup/PTM_PAGE
+2. Check the open issues that are available to the community and let us know you will tackle it
+3. When you finish, open up the pull request and we will review it as soon as possible
+4. If everything is fine, it will be merged and you will see it on the PTM website soon
+5. Later, your name will be featured in the Contributors section
```
Comment (FabioCB) [inline README.md]: Shouldn't the owner of the PR add the name to the List?

Comment (filipemcarvalho) [reply README.md]: Well played

File: README.md (hunk 2 above)
Comment (filipemcarvalho) [reply README.md]: done

File: index.md (hunk 1 above)
Comment (filipemcarvalho) [reply index.md]: done

File: README.md (hunk 3 above)
Comment (filipemcarvalho) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+4. When you finish, open up the pull request
+5. Wait patiently for a review...
+6. If everything is fine, it will be merged and you will see it on the PTM website soon
+
+# How to use your github repository to test?
+
+1. Open the url of your fork of the repository (example: https://github.com/yourgithubusername/PTM_PAGE)
+2. Go to the settings of the repository
+3. Open the options and scroll until Github Pages
+4. Select the theme (Jekyll theme is the one used by the website)
```
Comment (FabioCB) [inline README.md]: We are using Cayman Theme

File: README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+5. Select where your site is published - should be under something like https://github.com/yourgithubusername/PTM_PAGE
```
Comment (FabioCB) [inline README.md]: https://yourgithubusername.github.io/PTM_PAGE/

**Review by FabioCB — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (filipemcarvalho) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (filipemcarvalho) [reply README.md]: done

**Review by FabioCB — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+# How to contribute
+
+If you want to contribute to a project and make it better, your help is very welcome.
+
+1. Fork the repository https://github.com/portotestersmeetup/PTM_PAGE
+2. Check the open issues that are available to the community and let us know you will tackle it
+Note: **We will only accept PRs previously discussed in the issues section**
+3. Add your name in the Contributors section in this PR - if it is not there already
+4. When you finish, open up the pull request
+5. Wait patiently for a review...
```
Comment (FabioCB) [inline README.md]: remove ...

**Review by FabioCB — CHANGES_REQUESTED** [review]

**Pull request merged**